### PR TITLE
Add a report about Presto conference Tokyo 2020

### PR DESCRIPTION
**Pull request by toru-takahashi**
I would like to post a report about Presto Conference Tokyo 2020.

Thank you!

### CODE REVIEW

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "A Report about Presto Conference Tokyo 2020 Online"
+author: Toru Takahashi, Treasure Data
+excerpt_separator: <!--more-->
+---
+
+On Nov 11th, 2020, Japan Presto Community has held the 2nd Presto Conference
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:8]: remove "has"

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "A Report about Presto Conference Tokyo 2020 Online"
+author: Toru Takahashi, Treasure Data
+excerpt_separator: <!--more-->
+---
+
+On Nov 11th, 2020, Japan Presto Community has held the 2nd Presto Conference 
+with welcoming Martin Traverso and Brian.
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:9]: remove "with"

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+The conference was hosted at Youtube Live.
+This article is the summary of the conference aiming to share their great talks. 
+
+<!--more-->
+
+# Presto Community Updates
+
+First of all, Martin introduce recent Presto updates in these days.
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:17]: introduced

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+with welcoming Martin Traverso and Brian.
+The conference was hosted at Youtube Live.
+This article is the summary of the conference aiming to share their great talks. 
+
+<!--more-->
+
+# Presto Community Updates
+
+First of all, Martin introduce recent Presto updates in these days. 
+It covers rencent changes and enhancements achieved by the community activities.
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:18]: recent

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+- Update / Merge (https://github.com/prestosql/presto/issues/3325)
+- Materialized Views (https://github.com/prestosql/presto/pull/3283)
+- Dynamically resolved functions
+- Optimized Parquet reader
+
+In addition, at Q&A, he suggests new developers who want to contribute to PrestoSQL 
+to check "good first issue" tag on Github. The tag is a good first step for a new joiner to contribute. 
+Ref. [link](https://github.com/prestosql/presto/labels/good%20first%20issue)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=189" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:30]: Use the youtube tag as used in other posts

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+To make attendees get used to Presto Community, Martin provided a guide for walking around Presto community. 
+He gives us their team's principles about the Presto community, and talk about their education strategy for new Presto users.
+I would like to quote the pricinpals here.
+
+- We are passionate about open source
+- We help others be succesfful with what we create
+- We create robust long-lasting software
+- We are egalitarian (nobody is more important than the other)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=1429" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:43]: youtube tag

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+# Support Presto as a feature of SaaS
+
+Then, Satoru Kamikaseda, Technical Support Engineer at Treasure Data, provides an overview of how Treasure Data supports Presto in their service. 
+Presto is heavily used to support many enterprise use cases as a customer data platoform, 
+and it is becoming the hub component processing high throughput workload from many kinds of clients such as Spark, ODBC and JDBC.
+
+He describes a stats of Presto inquieries on their platform, and how to support each cases.
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:51]: just use "He described statistics about Presto queries on their platform..  "

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=1429" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+# Support Presto as a feature of SaaS
+
+Then, Satoru Kamikaseda, Technical Support Engineer at Treasure Data, provides an overview of how Treasure Data supports Presto in their service. 
+Presto is heavily used to support many enterprise use cases as a customer data platoform, 
+and it is becoming the hub component processing high throughput workload from many kinds of clients such as Spark, ODBC and JDBC.
+
+He describes a stats of Presto inquieries on their platform, and how to support each cases. 
+In the stats, 1/3 is any investigation of job failure and query result, 1/3 is a request to help their client's sql,
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:52]: "SQL"

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+# Support Presto as a feature of SaaS
+
+Then, Satoru Kamikaseda, Technical Support Engineer at Treasure Data, provides an overview of how Treasure Data supports Presto in their service. 
+Presto is heavily used to support many enterprise use cases as a customer data platoform, 
+and it is becoming the hub component processing high throughput workload from many kinds of clients such as Spark, ODBC and JDBC.
+
+He describes a stats of Presto inquieries on their platform, and how to support each cases. 
+In the stats, 1/3 is any investigation of job failure and query result, 1/3 is a request to help their client's sql, 
+and others are a sort of notifications to their clients and performance investigation. 
+His talk must be useful for any SaaS companies that provides a query engine to thier clients to learn how hard to support a distibuted query engine.
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:54]: their

Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:54]: how difficult it is to support

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+We could learn how to use Presto with AWS including Presto on EMR, Presto on EC2, Presto by Athena and AWS Glue.
+Noritaka Sekiyama, Sr. Big Data Architect at Amazon Web Service, Japan, also shares a comparison of Presto on AWS (EC2, EMR, Athena). 
+If you are a new to Presto, his talk gives you an insight to choose your first Presto environement.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/GR6e3dfKKJ8w4c" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/SatoruKamikaseda/support-presto-as-a-feature-of-saas" title="Support Presto as a feature of SaaS" target="_blank">Support Presto as a feature of SaaS</a> </strong> from <strong><a href="https://www.slideshare.net/SatoruKamikaseda" target="_blank">SatoruKamikaseda</a></strong> </div>
+
+# Presto @ LINE 2020
+
+LINE is the biggest company providing the mobile communication tool in Japan (say WhatsApp in Japan). HYuya Ebihara, one of Presto mainteners,
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:68]: maintainers

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+If you are a new to Presto, his talk gives you an insight to choose your first Presto environement.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/GR6e3dfKKJ8w4c" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/SatoruKamikaseda/support-presto-as-a-feature-of-saas" title="Support Presto as a feature of SaaS" target="_blank">Support Presto as a feature of SaaS</a> </strong> from <strong><a href="https://www.slideshare.net/SatoruKamikaseda" target="_blank">SatoruKamikaseda</a></strong> </div>
+
+# Presto @ LINE 2020
+
+LINE is the biggest company providing the mobile communication tool in Japan (say WhatsApp in Japan). HYuya Ebihara, one of Presto mainteners, 
+gives us how they improve Presto at their platform since they presented in the previous conference (https://prestosql.io/blog/2019/07/11/report-for-presto-conference-tokyo.html).
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:69]: use the internal link reference to link to old post

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+introduces the latest features of AWS Athena and performance tuning tips. It must be helpful for developers who tied to AWS to explore Amazon Athena.
+
+<script async class="speakerdeck-embed" data-id="92a399aad5344df197279cd4195d9464" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
+
+# Presto Cassandra Connector Hack at Repro
+
+Repro provides Customer Engagement Platform that enables companies to personalize their communication strategies with the right message at the right time to drive better retention and lifetime value. 
+They use Presto for a segmentation backend system in their service to make a list of audiences with a certain condition. 
+
+![](https://docs.repro.io/en/_images/segmentation-filter.png)
```
Comment (mosabua) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:87]: Better not use an external image like that .. maybe add it to the PR in a suitable size or link to it.

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 11 above)
Comment (ebyhr) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:69]: We can use `[the previous conference]({% post_url 2019-07-11-report-for-presto-conference-tokyo %})`

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+We could learn how to upgrade Presto from 314 to 339 and how they resolved issues through Presto upgrade.
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vS2QdQjhLsiSuVdWlEmT23ixqoZXkRrKKMRGa1hrZHg65OpcH18RpzARotOMYvIBSwP57lPPAHkUQOx/embed" frameborder="0" width="1280" height="749" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+# Dive into Amazon Athena - Serverless Presto, 2020
+
+Makoto Kawamura, Solution Architect at Amazon Web Service Japan, 
+introduces the latest features of AWS Athena and performance tuning tips. It must be helpful for developers who tied to AWS to explore Amazon Athena.
+
+<script async class="speakerdeck-embed" data-id="92a399aad5344df197279cd4195d9464" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
```
Comment (ebyhr) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:80]: Current link looks wrong. 
```
<iframe src="//www.slideshare.net/slideshow/embed_code/key/kWzJ1XqR96A9di" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/ssuserca76a5/aws-presto" title="AWS で Presto を徹底的に使いこなすワザ" target="_blank">AWS で Presto を徹底的に使いこなすワザ</a> </strong> from <strong><a href="https://www.slideshare.net/ssuserca76a5" target="_blank">Noritaka Sekiyama</a></strong> </div>
```

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+
+# Presto @ LINE 2020
+
+LINE is the biggest company providing the mobile communication tool in Japan (say WhatsApp in Japan). HYuya Ebihara, one of Presto mainteners, 
+gives us how they improve Presto at their platform since they presented in the previous conference (https://prestosql.io/blog/2019/07/11/report-for-presto-conference-tokyo.html). 
+Their Presto usage significantly increases from 2019. Num of Presto workers from 100 to 300 and Num of daily queries reaches to 50,000 queries from 20,000 queries. 
+We could learn how to upgrade Presto from 314 to 339 and how they resolved issues through Presto upgrade.
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vS2QdQjhLsiSuVdWlEmT23ixqoZXkRrKKMRGa1hrZHg65OpcH18RpzARotOMYvIBSwP57lPPAHkUQOx/embed" frameborder="0" width="1280" height="749" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```
Comment (ebyhr) [inline _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:73]: Sorry, the iframe size I shared before was too large. 
```suggestion
<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vS2QdQjhLsiSuVdWlEmT23ixqoZXkRrKKMRGa1hrZHg65OpcH18RpzARotOMYvIBSwP57lPPAHkUQOx/embed" frameborder="0" width="595" height="485" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 5 above)
Comment (ebyhr) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:30]: We can use
```
{% youtube NxDBBEA67Ws %}
```

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 6 above)
Comment (ebyhr) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:43]: @mosabua Does youtube tag support start-time option?

**Comment by toru-takahashi** [comment]
Thank you for your review. I'll fix based on these suggestions.

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 12 above)
Comment (toru-takahashi) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:87]: let me remove it.

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 13 above)
Comment (toru-takahashi) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:80]: thanks.

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 11 above)
Comment (toru-takahashi) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:69]: thanks.

File: _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md (hunk 6 above)
Comment (toru-takahashi) [reply _posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md:43]: Since their talks are in series, let me remove this one.

**Comment by toru-takahashi** [comment]
Fixed the suggestions.

**Comment by toru-takahashi** [comment]
@cla-bot check

**Comment by toru-takahashi** [comment]
Hmm, sorry this is not Presto related. But, somehow i couldn't fix this issue > Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: Toru Takahashi.

```
 % git config --list | grep email
user.email=takahashi@treasure-data.com
```

I've confirmed there is no issue with my CLI and Github console. Do you know where I should fix?

**Comment by ebyhr** [comment]
The message comes from the 1st commit https://github.com/prestosql/prestosql.io/pull/132/commits. 
Could you fixup the 2nd commit and then update commit author?
```
git rebase -i HEAD~2

# Below message will appear
pick b426d5a Add a report about Presto conference Tokyo 2020
pick 3beda58 Update 2020-11-21-a-report-about-presto-conference-tokyo-2020.md

# Replace the 2nd pick with fixup and save
pick b426d5a Add a report about Presto conference Tokyo 2020
fixup 3beda58 Update 2020-11-21-a-report-about-presto-conference-tokyo-2020.md

# Update commit author
git commit --amend --author="Toru Takahashi <toru-takahashi@users.noreply.github.com>"
```

Also, could you please submit CLA to https://github.com/prestosql/cla if you haven't sent it?

**Comment by toru-takahashi** [comment]
Oops. I wrongly updated it... let me cleaned up.

**Comment by toru-takahashi** [comment]
I fixed my commit itself. I'll fix the account setting for each commit in the next. Sorry for the delay in my update.

**Comment by toru-takahashi** [comment]
I fixed up my commits. And, I think I've sent signed CLA to cla@prestosql.io.
Could you help me with the next step?

**Comment by toru-takahashi** [comment]
https://prestosql.slack.com/archives/CFP4810CV/p1607540585012400?thread_ts=1607539593.010600&cid=CFP4810CV

As Martin said, let me skip CLA bot since it's not nessesary.

> For blog posts, we don’t need a CLA. Unfortunately, it’s not possible to limit the CLA bot to specific repositories or parts of repositories, so we get those “false positive” notifications.

Could I ask @ebyhr or @mosabua  to review/merge if this PR is ok?

**Comment by mosabua** [comment]
I don't have merge rights. Maybe @bitsondatadev should have a look next and then @martint can have a look and merge.

**Pull request merged**